l10n_it_pec_messages/model/mail_thread.py

```diff
@@ -154,9 +162,19 @@ def message_parse(
                 # I'm going to set this message as notification of the original
                 # message and remove the message_id of this message
                 # (it would be duplicated)
-                context['main_message_id'] = msg_ids[0]
-                context['pec_type'] = daticert_dict.get('pec_type')
-            del msg_dict['message_id']
+                # before deletion check if this message is prensent and linked with
+                # main massage id, if false remove message_id else no
+                chk_msgids = message_pool.search(
+                    cr, uid,
+                    [
+                        ('pec_msg_id', '=', daticert_dict['pec_msg_id']),
+                        ('pec_type', '=',  daticert_dict.get('pec_type'))
+                    ],
+                    context=context)
+                if not chk_msgids:
```

Comment:
@angedras right. I noticed it yesterday.
Morevoer, due to https://github.com/archetipo/l10n-italy/pull/19#issuecomment-78243311 this branch should be deleted and a new one created